extensions/amp-ad-network-doubleclick-impl/sra.md

```diff
@@ -21,11 +21,12 @@ Enabling SRA allows a publisher to make a single request for all ad slots on the
 In order to use this feature, add the following meta tag to the head of the AMP page:
 `<meta name=”amp-ad-doubleclick-sra”/>`
 
-Note that SRA is not available in the following cases:
-1. If the AMP page is not served from a valid AMP cache
-2. If publishers use [`remote.html`](https://github.com/ampproject/amphtml/blob/master/ads/README.md#1st-party-cookies)
-3. The ad refresh feature is incompatible with SRA
-4. Publishers don't use the amp-ad attribute [`useSameDomainRenderingUntilDeprecated`](https://github.com/ampproject/amphtml/blob/master/ads/google/doubleclick.md#temporary-use-of-usesamedomainrenderinguntildeprecated)
+Note that SRA is only available in the following cases:
+1. The AMP page is served from a valid AMP cache
+2. The publisher does not use [`remote.html`](https://github.com/ampproject/amphtml/blob/master/ads/README.md#1st-party-cookies)
+3. The publisher uses the amp-ad attribute [`useSameDomainRenderingUntilDeprecated`](https://github.com/ampproject/amphtml/blob/master/ads/google/doubleclick.md#temporary-use-of-usesamedomainrenderinguntildeprecated)
```

Comment:
I'm confused.  This says SRA depends on the publisher using `useSameDomainRenderingUntilDeprecated`, but that's ignored now, right?  And this PR removes the doc for that.  Does this mean SRA is no longer available?

Reply:
(The link should probably be to https://github.com/ampproject/amphtml/blob/master/ads/google/doubleclick.md#temporary-use-of-usesamedomainrenderinguntildeprecated-until-march-29-2018 and not https://github.com/ampproject/amphtml/blob/master/ads/google/doubleclick.md#temporary-use-of-usesamedomainrenderinguntildeprecated but neither link will work once this PR is in since you're deleting that section)

Reply:
Removed that bit.